owncloudApp/src/main/res/values/styles.xml

```diff
@@ -96,6 +96,10 @@
         <item name="windowNoTitle">true</item>
     </style>
 
+    <!-- Alert Dialog title -->
+    <style name="Theme.ownCloud.AlertDialog.WindowTitle" parent="TextAppearance.Material3.TitleMedium">
+    </style>
```

Comment:
This is not needed anymore in case you apply what I suggest in the previous comment 👍

Reply:
Thanks for the suggestions. Removed as suggested. However, I would like to clarify the following points.

 1) Isn't it better to move eventually to a material3 day-night theme rather than using a device default theme?
2) I noticed the app doesn't have any effect on Day/night mode changes. In that case, using components from the device's default theme may change based on the device theme right?

Reply:
Sure, Material3 will be a future aim in our app 😃. We're thinking of including big UI changes soon, so all of these will be taken into account, for the moment the best is keeping it as simple as possible